modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Ga4gh_conf.pm

```diff
@@ -0,0 +1,153 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::Ga4gh_conf;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Production::Pipeline::PipeConfig::Base_conf');
+
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use Bio::EnsEMBL::Hive::Version 2.5;
+use File::Spec;
+
+sub default_options {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::default_options()},
+
+        ## General parameters
+        'release'                => $self->o('ensembl_release'),
+        'pipeline_name'          => "ga4gh_checksum_generator_".$self->o('ensembl_release'),
+        'web_email'              => '',
+        'sequence_types'         => [],
+	'run_all'                => 0, 
+        ## 'job_factory' parameters
+        'species'                => [],
+        'antispecies'            => [],
+        'division'               => [],
+        'dbname'                 => undef,
+	#checksum params
+	'sequence_type'         => [],
+	'hash_type'             => [],
+    };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+
+  return [
+    @{$self->SUPER::pipeline_create_commands}
+  ];
+}
+
+# Ensures output parameters gets propagated implicitly
+sub hive_meta_table {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::hive_meta_table},
+    'hive_use_param_stack' => 1,
+  };
+}
+
+sub pipeline_wide_parameters {
+  my ($self) = @_;
+
+  return {
+    %{$self->SUPER::pipeline_wide_parameters},
+    'pipeline_name' => $self->o('pipeline_name'),
+    'release'       => $self->o('release'),
+    'sequence_types'         => $self->o('sequence_type'),
+    'hash_types'             => $self->o('hash_type'),
+
+
+  };
+}
+
+sub resource_classes {
```

Comment:
Same comment about the resource class, it doesn't make any sense to add things here, better to put it at top level in hierarchy, so we can use this pipeline on SLURM